src/List/ListItem/ListItem.styles.ts

```diff
@@ -3,100 +3,46 @@ import styled, { css } from 'styled-components';
 export interface ListItemProps extends React.HTMLAttributes<HTMLDivElement> {
   disabled?: boolean;
   variant?: string;
-  radius?: string;
-  name?: string;
 }
 
 export const StyledListItem = styled.div<ListItemProps>`
+  cursor: pointer;
   color: ${p => p.theme.colors.grayDark};
   font-weight: 700;
   background-color: ${p => p.theme.colors.white};
   position: relative;
   padding: ${p => p.theme.spacing.xsmall};
+  :not(:last-child) {
+    border-bottom: 1px solid ${p => p.theme.colors.gray};
```

Comment:
Actually we've lost the opacity here on `border-bottom`, I added that previously to align with the design.